integration/images/private-workdir/Dockerfile

```diff
@@ -0,0 +1,20 @@
+#   Copyright The containerd Authors.
+
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+
+#       http://www.apache.org/licenses/LICENSE-2.0
+
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+FROM busybox
+
+RUN mkdir /workdir
+RUN chmod 700 /workdir
+RUN chown nobody /workdir
+WORKDIR /workdir
```

Comment:
github complaining about a missing new line at end of file...

Reply:
should have one now.